examples/kuka_iiwa_with_orientation.py

```diff
@@ -0,0 +1,227 @@
+import os
+import gymnasium as gym
+import numpy as np
+import casadi as ca
+from forwardkinematics.urdfFks.generic_urdf_fk import GenericURDFFk
+from scipy.spatial.transform import Rotation as R
+from urdfenvs.urdf_common.urdf_env import UrdfEnv
+from urdfenvs.robots.generic_urdf import GenericUrdfReacher
+from urdfenvs.sensors.full_sensor import FullSensor
+
+from mpscenes.goals.goal_composition import GoalComposition
+from mpscenes.obstacles.sphere_obstacle import SphereObstacle
+
+from fabrics.planner.parameterized_planner import ParameterizedFabricPlanner
+
+"""
+Experiments with Kuka iiwa
+If 
+    static_obst_dict = {
+        "type": "sphere",
+        "geometry": {"position": [0.3, -0.3, 0.5], "radius": 0.1},
+    }
+for the first obstacle, fabrics ends up in a local minima and is unable to recover from it. 
+This would be a good case to improve with imitation learning.
+"""
+
+def initalize_environment(render=True, nr_obst: int = 0):
+    """
+    Initializes the simulation environment.
+
+    Adds obstacles and goal visualizaion to the environment based and
+    steps the simulation once.
+    """
+    robots = [
+        GenericUrdfReacher(urdf="iiwa7.urdf", mode="acc"),
+    ]
+    env: UrdfEnv  = gym.make(
+        "urdf-env-v0",
+        dt=0.01, robots=robots, render=render
+    ).unwrapped
+    full_sensor = FullSensor(
+            goal_mask=["position", "weight"],
+            obstacle_mask=['position', 'size'],
+            variance=0.0
+    )
+    # Definition of the obstacle.
+    static_obst_dict = {
+        "type": "sphere",
+        "geometry": {"position": [0.3, -0.3, 0.3], "radius": 0.1}, #todo: IMPORTANT when z=0.5: fabrics becomes unstable/local minima
+    }
+    obst1 = SphereObstacle(name="staticObst", content_dict=static_obst_dict)
+    static_obst_dict = {
+        "type": "sphere",
+        "geometry": {"position": [-0.7, 0.0, 0.5], "radius": 0.1},
+    }
+    obst2 = SphereObstacle(name="staticObst", content_dict=static_obst_dict)
+    # Definition of the goal.
+    rot_matrix = np.array([[-0.339, -0.784306, -0.51956],
+                      [-0.0851341, 0.57557, -0.813309],
+                      [0.936926, -0.23148, -0.261889]])
+    r = R.from_matrix(rot_matrix)
+    quat_list = list(r.as_quat())
+    sub_goal_0_quaternion = [float(quat_i) for quat_i in quat_list]
```

Comment:
Is this really needed? I think you only need it in the main, right?

Reply:
indeed, this is no longer needed!